.github/workflows/ci.yml

```diff
@@ -41,3 +28,54 @@ jobs:
         make
         sudo make install
         ctest
+
+  build_windows:
+    runs-on: windows-latest
+    steps:
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          update: true
+          install: git mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-openssl
+      - name: Add mingw64 to PATH
+        run: |
+          # First make sure python would resolve to the windows native python, not mingw one
+          echo "C:/msys64/mingw64/bin" | Out-File -Append -FilePath $env:GITHUB_PATH -Encoding utf8
+          echo "${{ env.pythonLocation }}" | Out-File -Append -FilePath $env:GITHUB_PATH -Encoding utf8
+      - uses: actions/checkout@v1
+      - name: Build and install mgconsole
+        run: |
+          mkdir build
+          cd build
+          cmake -G "MinGW Makefiles" ..
+          make
+          make install
+      - name: Save mgconsole Windows build
+        uses: actions/upload-artifact@v2
+        with:
+          name: "mgconsole Windows build"
+          path: build/src/mgconsole.exe
+          #ToDo(the-joksim):
+          #  - add Windows e2e test
+
+  build_apple:
+    strategy:
+      matrix:
+        platform: [macos-10.15]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Set-up repository
+        uses: actions/checkout@v2
+      # NOTE: CI can't execute end2end tests because there is no way to run
+      #       Memgraph on CI MacOS machines.
+      - name: Build mgconsole
+        run: |
+          mkdir build
+          cd build
+          cmake -DOPENSSL_ROOT_DIR="$(brew --prefix openssl)" ..
```

Comment:
Build in the release mode by addding `-DCMAKE_BUILD_TYPE=Release ..`

Reply:
Will do.